spec/spec_helper.rb

```diff
@@ -38,7 +38,7 @@
 
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
-
+require 'shoulda/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.